packages/ckeditor5-html-support/docs/features/full-page-html.md

```diff
@@ -0,0 +1,60 @@
+---
+category: features-html
+order: 50
+modified_at: 2023-05-15
+---
+
+# Full page HTML
+
+{@snippet features/general-html-support-source}
+
+The Full Page HTML feature allows you to preserve the whole HTML page in the editor data. With this plugin, you can enable certain HTML tags in your editor. But, unlike GHS, which focuses on tags inside the content, this feature keeps the markup that's usually not visible to the user, like the wrapping `<html></html>` tag.
```

Comment:
```suggestion
Thanks to the full page HTML feature you can use CKEditor 5 to edit entire HTML pages (from `<html>` to `</html>`), including the page metadata. While the {@link features/general-html-support General HTML Support} feature focuses on elements inside the content (the document's `<body>`), this feature enables markup mostly invisible to the end user.
```
What do you think?

Reply:
Nicely rephrased - the description is more clear now.